e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -208,7 +195,7 @@ def run_diag(parameter):
                             ),
                             parameter.output_file + ".json",
                         )
-                        print(f"Metrics saved in: {fnm}")
+                        logger.info(f"Metrics saved in: {fnm}")
 
```

Comment:
I would move lines 200-215 outside of the conditional (line 289) and delete lines 268-283 since the logic is the same.